docs/static/filebeat_modules/nginx/error/pipeline.conf

```diff
@@ -0,0 +1,29 @@
+input {
+  beats {
+    # The port to listen on for filebeat connections.
+    port => 5044
+    # The IP address to listen for filebeat connections.
+    host => "0.0.0.0"
+  }
+}
+filter {
+   grok {
+      match => { "message" => ["%{DATA:[nginx][error][time]} \[%{DATA:[nginx][error][level]}\] %{NUMBER:[nginx][error][pid]}#%{NUMBER:[nginx][error][tid]}: (\*%{NUMBER:[nginx][error][connection_id]} )?%{GREEDYDATA:[nginx][error][message]}"] }
+      remove_field => "message"
+   }
+   mutate {
+      rename => { "@timestamp" => "read_timestamp" }
```

Comment:
removing the `@timestamp` field here will cause the elasticsearch output to crash when resolving the index name `%{[@metadata][beat]}-%{+YYYY.MM.dd}`